packages/web-app-files/src/components/AppBar/CreateAndUpload.vue

```diff
@@ -135,19 +112,46 @@ import { useAppDefaults } from 'web-pkg/src/composables'
 
 import { DavProperties, DavProperty } from 'web-pkg/src/constants'
 
-import FileDrop from './Upload/FileDrop.vue'
+// TODO: Simplify to one UploadButton component and fill from here
```

Comment:
Comment can be deleted since the "multiple files or one folder" upload topic is an OS-level restriction AFAIK

Reply:
Clarified this is about cleaning up the two {File,Folder}Upload.vue components since they hold quite a bit of duplication (and hence could be filled using props/attrs)